### DISCUSSION TITLE
Force fit the active terminal size

### DISCUSSION THREAD

**Discussion by izadi** [zellij-org/zellij#3816] [Q&A]
It seems to me that if I'm connected to a session from multiple terminals at the same time, Zellij is sized to the minimum size of the terminals. This could be problematic if for example I'm connected to a session from a small screen device (like a phone). If I connect to the same session from my laptop, Zellij would still be shown to me with the size appropriate for the small screen and if I don't have access to the small screen to detach it, I'd be stuck with the small size on my laptop.

Is there a way to request Zellij to resize to my active terminal? (In GNU Screen I can use the [fit](https://www.gnu.org/software/screen/manual/screen.html#Fit) command to achieve this). Alternatively, as a workaround, is there a way to detach another Zellij instance from a session?

**Comment by Lockszmith-GH** [comment]
The [session-manager](https://zellij.dev/documentation/session-manager-alias) has a ~~"close all other sessions"~~ "disconnect others" option (<kbd>Ctrl</kbd>+<kbd>x</kbd> in the session manager)

**Reply by izadi** [reply]
Great. That answer my workaround question (though "close all other sessions" is not an accurate description of what it does. It disconnects all other users of the same session). Thanks!

**Comment by gunererd** [comment]
It used to fit the active terminal size just a couple of weeks ago, I'm sure of it.

Now it's not resizing session regarding to focused terminal. I tried various terminals but it seems not related.
